Engine-Editor/GameJam/Assets/Scripts/GameJam/Source/Players/CirclePlayer.cs

```diff
@@ -48,6 +48,13 @@ public void OnCollisionExit(Collision2D collision)
             collision.otherEntity.GetComponent<SpriteRendererComponent>().color = Color.brown;
         }
 
+        public void OnKeyPress(KeyCode keyCode)
+        {
+            DebugConsole.Log("Key pressed "+keyCode, DebugConsole.LogType.Info);
+            // Input.SetClipboardContent("Nacisnales " + keyCode.ToString());
```

Comment:
Delete comments if the purpose of their existance was testing.

Reply:
Done